src/dnx/dnx.vcxproj

```diff
@@ -98,7 +98,7 @@
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions);$(Defines)</PreprocessorDefinitions>
-      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
+      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
```

Comment:
@davidfowl I seem to recall that on NanoServer and above (including full flavors) we only have c:\windows\system32\msvcrt.dll (the built in CRT). I am fairly certain that we don't have a debug version preinstalled i.e. no msvcrtd.dll